templates/blazor-base/csharp/default/GettingStarted.txt

```diff
@@ -1,12 +1,25 @@
 Welcome to the Teams Toolkit! Here are some tips to get started building your app.
 
+Prerequisites
+----------
+
+If the project contains Bot capability, you will need to setup Ngrok(https://ngrok.com/) in order to tunnel from the Teams client to localhost.
+
+After install Ngrok please run the command in Command Prompt: ngrok http 2544
+
 
 Quick start
```

Comment:
Before you run or Start Debugging for the first time, follow these steps:

1. If your project only contains a Tab capability, skip to step #3.

2. For projects that contain a Bot capability, first use a Command Prompt to run this command: ngrok http 2544.

3. Select the Project > TeamsFx > Configure Microsoft Teams app menu in Visual Studio. When prompted, sign in using your M365 account.
 
4. Select the Debug > Start Debugging menu in Visual Studio or press F5. Visual Studio will launch the app inside Microsoft Teams using a browser.

5. Once Microsoft Teams is loaded, select the Add button to install your app in Teams.